frappe/installer.py

```diff
@@ -75,12 +77,18 @@ def _new_site(
 		db_port=db_port,
 		no_mariadb_socket=no_mariadb_socket,
 	)
-	apps_to_install = (
+	apps_to_install = set(
```

Comment:
using a set here because if someone passes a list of app names in `install_apps` param and it already had frappe then it will try to reinstall frappe.